service/src/main/java/com/fitgoal/service/impl/NotificationServiceImpl.java

```diff
@@ -65,4 +68,15 @@ private void registerEvent(String serviceName, String event) {
                 .date(LocalDateTime.now()).build();
         auditDao.create(auditDto);
     }
+    private URI buildUri(String verificationLink) {
+        try {
+            return new URIBuilder()
+                    .setScheme("http")
```

Comment:
please extract it into a separate config file or reuse MailerConfiguration fields. One of the ways how to do that, is to inject MailerConfiguration in the constructor and store scheme/host/port in the fields.

Reply:
Done